fetch.bs

```diff
@@ -5063,14 +5063,13 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
 
      <li>
       <p>If <var>request</var>'s <a for=request>body</a> is non-null, and <var>request</var>'s
-      <a for=request>body</a>'s <a for=body>source</a> is null, the user agent may have a buffer of
-      <a>implementation-defined</a> size and store a part of <var>request</var>'s
-      <a for=request>body</a> to the buffer. The buffer is used to resend <var>request</var>'s
-      <a for=request>body</a> to the network. If the user agent sent more than the buffer size and
-      is required to resend, return a <a>network error</a>.
+      <a for=request>body</a>'s <a for=body>source</a> is null, the user agent may have a buffer up
+      to 64k bytes and store a part of <var>request</var>'s <a for=request>body</a> to the buffer.
```

Comment:
Please write this out as we do elsewhere, "64 kibibytes" (I'm assuming we don't want literal kilobytes in this instance).